basic-data-structures/copy-array-items-using-slice.js

```diff
@@ -0,0 +1,8 @@
+function forecast(arr) {
+  // Only change code below this line
+  let newArray = arr.slice(2, 4);
```

Comment:
you dont need to create a new variable, only return the slice
return arr.slice....